### PR TITLE
Add players to leaderboard

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
### Project requirements

-  Implement the "Refresh" button (receiving data from the API and parsing the JSON).
-  Implement the form "Submit" button (sending data to the API).
-  Use async and await JavaScript features to consume the API.
-  No styling is required.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
All linters have passed. No styling is required for this project.

**Comment by Ugboaja-Uchechi** [comment]
Worked on project requirements to the best of my knowledge

File: src/index.html [hunk 1]
```diff
@@ -16,14 +16,14 @@ <h3>Recent Scores <span><button type="reset">Refresh</button></span></h3>
         <ul class="score-list">
```
Comment (rloterh) [inline src/index.html:16]: - [ ] OPTIONAL 
The _ReadMe_ for this project is **missing**. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, its contents need to reflect this project,  and may likely feature on your portfolio. Kindly ensure to add a ReadMe. You may use [this template](https://github.com/microverseinc/readme-template) to craft a professional ReadMe but remember to customize it to your project.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Stephanie,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- JavaScript promise feature well implemented
- Data is sent and received over the API successfully
- No linter errors

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review:_
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**